### ISSUE TITLE
Deploy on AWS us-east-1 fails...

### ISSUE THREAD

**Issue by kristvanbesien** [openshift/openshift-ansible-contrib#310]
Deploying on AWS with these values:
```
Configured values:
	stack_name: openshift-infra
	ami: ami-a33668b4
	region: us-east-1
	master_instance_type: m4.large
	node_instance_type: t2.medium
	app_instance_type: t2.medium
	bastion_instance_type: t2.micro
	keypair: aws
	create_key: no
	key_path: /dev/null
	create_vpc: yes
	vpc_id: None
	private_subnet_id1: None
	private_subnet_id2: None
	private_subnet_id3: None
	public_subnet_id1: None
	public_subnet_id2: None
	public_subnet_id3: None
	byo_bastion: no
	bastion_sg: /dev/null
	console port: 443
	deployment_type: openshift-enterprise
	openshift_sdn: openshift-ovs-subnet
	public_hosted_zone: ocp.vanbesien.net
	app_dns_prefix: apps
	apps_dns: apps.ocp.vanbesien.net
	rhsm_user: kvanbesi@redhat.com
	rhsm_password: *******
	rhsm_pool: 8a85f98144844aff014488d059f61625
	containerized: False
	s3_bucket_name: ocp-krist-registry-bucket
	s3_username: openshift-infra-s3-openshift-user
	github_client_id: *******
	github_client_secret: *******
	github_organization: ocp-krist

```

So using the standard AMI, and us-east-1

I get the following error:

```
TASK [cloudformation-infra : Create Greenfield Infrastructure] *****************
fatal: [localhost]: FAILED! => {
    "changed": false, 
    "failed": true
}

MSG:

Template error: Fn::Select  cannot select nonexistent value at index 2

```

What would be the reason?

Doing this in eu-west-1, with a different ami worked...

**Comment by cooktheryan** [comment]
I wonder if it was an error when trying to select an AZ that wasn't vpc classic

**Comment by cooktheryan** [comment]
This should be resolved when #315 is merged. Which should be tomorrow.
